pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -355,7 +353,7 @@ func generatePortMappings(clusterIPFamily config.ClusterIPFamily, portMappings .
 		if strings.HasSuffix(hostPortBinding, ":0") {
 			hostPortBinding = strings.TrimSuffix(hostPortBinding, "0")
 		}
-		args = append(args, fmt.Sprintf("--publish=%s:%d/%s", hostPortBinding, pm.ContainerPort, protocol))
+		args = append(args, fmt.Sprintf("--publish=%s:%d/%s", hostPortBinding, pm.ContainerPort, strings.ToLower(protocol)))
```

Comment:
aside: this really seems like a bug in podman.

Reply:
Already fixed on master, but it slipped into a few published releases

Reply:
docker accepts uppercase AND this is the canonical value in CRI, which is why we're using uppercase https://github.com/kubernetes/cri-api/blob/34366a3c19379c566a82895ce1bcb75d9a502fac/pkg/apis/runtime/v1alpha2/api.pb.go#L58

Reply:
ack, thanks